sdk/nodejs/introspector/test/testdata/state/state.ts

```diff
@@ -0,0 +1,50 @@
+import { fct, object, Container, dag } from '@dagger.io/dagger'
+
+/**
+ * Alpine module
+ */
+@object
+export class Alpine {
+    private version = "3.16.2"
+
+    protected user = "root"
+
+    /**
+     * packages to install
+     */
+    public packages: string[]
+
+    ctr: Container
+
+    /**
+     * Returns a base Alpine container
+     * @param version version to use (default to: 3.16.2)
+     */
+    @fct
+    base(version?: string): Alpine {
```

Comment:
Why not put the default in the signature? 
```suggestion
    base(version: string = "3.16.2"): Alpine {
```
Then you'll be able to register the arg's [`defaultValue`](https://docs.dagger.io/current/sdk/nodejs/reference/classes/api_client_gen.FunctionArg#defaultvalue), which will show in the CLI automatically.

If that value needs to be shared in other places it can be made a constant.

Reply:
Not actually resolved but can be in follow-up though. 🙂